src/LinearOperators.jl

```diff
@@ -193,6 +193,33 @@ function LinearOperator(nrow :: Int, ncol :: Int,
   LinearOperator{T,F1,F2,F3}(nrow, ncol, symmetric, hermitian, prod, tprod, ctprod)
 end
 
+"""
+    LinearOperator(type, nrow, ncol, symmetric, hermitian, prod,
+                    [tprod=nothing,
+                    ctprod=nothing])
+
+Construct a linear operator from functions where the type is specified as the first argument.
+Notice that the linear operator does not enforce the type, so unexpected behavior can
+happen. For instance,
+```
+A = [im 1.0; 0.0 1.0] # Complex matrix
+op = LinearOperator(Float64, 2, 2, false, false, v->A*v, u->transpose(A)*u, w->A'*w)
+Matrix(A) # raises InexactError because tries to store operator on Float64 matrix
```

Comment:
When is the InexactError raised exactly? When you try to convert `op` to a matrix?